github/github.go

```diff
@@ -28,9 +28,11 @@ import (
 )
 
 const (
+	packageVersion = "45.2.0"
+
 	defaultBaseURL = "https://api.github.com/"
 	uploadBaseURL  = "https://uploads.github.com/"
-	userAgent      = "go-github"
+	userAgent      = "go-github" + "/" + packageVersion
```

Comment:
What do you think of renaming this to `defaultUserAgent`, which is more representative of what it is?